### DISCUSSION TITLE
Edit tweets / view edited tweets

### DISCUSSION THREAD

**Discussion by dnestico** [robertodoering/harpy#752] [Ideas]
Twitter announced today they're testing an edit tweet button and it looks like they'll be adding the ability to do the same to the new API soon too starting with being able to view tweets that have been edited already 👀 

Feel like this is a must-add for harpy in the future for many reasons, at least to be able to see already edited tweets is very important.

https://twitter.com/TwitterDev/status/1565362990509015040

<img width="595" alt="Screen Shot 2022-09-01 at 12 54 03 PM" src="https://user-images.githubusercontent.com/3034555/187969891-26d1197e-baa0-4089-969b-dd2f57b74a4d.png">

**Comment by robertodoering** [comment]
Nice! 🚀

**Reply by dnestico** [reply]
👀 https://trello.com/c/VBJPGj9Q

**Reply by robertodoering** [reply]
Thanks for sharing 🚀  nice to see that it also supports the old api

**Reply by robertodoering** [reply]
I've created a new issue for it: #786
